src/storage/types.rs

```diff
@@ -142,6 +144,10 @@ impl PessimisticLockRes {
                     (v.unwrap_or_default(), is_not_found)
                 })
                 .unzip(),
+            PessimisticLockRes::Existence(mut vals) => {
+                vals.iter_mut().for_each(|x| *x = !*x);
+                (vec![], vals)
```

Comment:
The returned result formats are diffrent for `value return` and `existence check`, would this confuse the `PessimisticLockResponse` users?

Reply:
Who would be confused?

Reply:
🤔 Sorry I didn't understand... This behavior only happens when `return_value` flag in the request is not set but `check_existence` is set, and when setting the flag, the user should know what (s)he is doing.

Reply:
The two fields `values` and `not_founds` in `PessimisticLockResponse` would be used by both `return_value` and `check_exisstence` flags after this change. The result formats are a bit different, the user needs to know exactly the difference. 
It dose not matter much and for simplicy seems this is the only way.